packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -3057,20 +3059,22 @@ describeIf(!process.env.TEST_SKIP_MONGODB)('MongoDB', () => {
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        - Model "users", field: "numberOrString1", chosen data type: "Json"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        - Type "UsersHobbies", field: "numberOrString2", chosen data type: "Json"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        - Type "UsersHobbiesObjects", field: "numberOrString3", chosen data type: "Json"
+      The following fields had data stored in multiple types. Either use Json or normalize data to the wanted type:
+        - model: users, field: numberOrString1, type: Json
 
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Run prisma generate to generate Prisma Client.
+      The following fields had data stored in multiple types. Either use Json or normalize data to the wanted type:
+        - composite type: UsersHobbies, field: numberOrString2, type: Json
+        - composite type: UsersHobbiesObjects, field: numberOrString3, type: Json
```

Comment:
This is intended from the new PR, which groups warnings with the same structure under the same headline.
I think this is an acceptable compromise.

Reply:
Here it would be better to have everything under the same headline, actually?
It looks like we have 2 (same) headline for fields with mixed data 
- one for models
- one for composite types

Reply:
See [schema-engine/connectors/schema-connector/src/warnings.rs line 341](https://github.com/prisma/prisma-engines/pull/3906/files#diff-3ecd59b1df3c163d7b9a810a0b11acab2435687ab41ca693f12366833485f366R341-R351)

Reply:
(not a big deal though)

Reply:
This could potentially be done, but it would be some sort of "ad hoc" solution that doesn't provide a considerable benefit to the user. I'd rather discard this and leave it as it is